tfs/src/main/java/hudson/plugins/tfs/ChangeSetReader.java

```diff
@@ -10,30 +10,31 @@
 import java.util.ArrayList;

```

Comment:
Ideally, you would remove the //CHECKSTYLE:OFF setting for the files that you modify and correct all the checkstyle warnings and errors in that file.

Reply:
imho a passing checkstyle run as part of the build is a minimal requirement for a contribution. Just get your IDE to format it correctly in an automated fashion. Or fix it manually if necessary

Reply:
ok, I removed the //CHECKSTYLE:OFF comments and fixed the fallout